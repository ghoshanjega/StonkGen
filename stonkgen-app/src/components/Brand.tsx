import React from 'react'

export const Brand = () => {
    return (
       <span>StonkGen</span>
    )
}
