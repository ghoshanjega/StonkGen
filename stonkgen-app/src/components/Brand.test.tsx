import React from 'react';
import { shallow } from 'enzyme';
import { Brand } from './Brand';


let wrapped = shallow(<Brand />);

describe('Brand Component', () => {
  it('should render the Brand name Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  it('renders the Brand text', () => { 
    expect(wrapped.find('span').text()).toEqual('StonkGen');
  });
});