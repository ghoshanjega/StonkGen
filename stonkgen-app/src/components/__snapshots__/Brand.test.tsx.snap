// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand Component should render the Brand name Component correctly 1`] = `ShallowWrapper {}`;
